### PR TITLE
p4 - btc consensus protocols

### PR DESCRIPTION
**Pull request by raftale**

### CODE REVIEW

File: BTC/P4-BTC Consensus Protocols.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+# Building a decentralized monetary system
```
Comment (wfnuser) [inline BTC/P4-BTC Consensus Protocols.md:1]: pleas update README.md to provide a related link to this file.

**Comment by raftale** [comment]
@wfnuser Got it. Resubmitted.

File: BTC/P4-BTC-Consensus-Protocols.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Build A Decentralized Monetary System
```
Comment (wfnuser) [inline BTC/P4-BTC-Consensus-Protocols.md:1]: Title should be P4 Consensus Protocols

Comment (raftale) [reply BTC/P4-BTC-Consensus-Protocols.md]: Yep, It is fixed.

**Review by wfnuser — APPROVED** [review]

**Pull request merged**